react-leaflet/index.d.ts

```diff
@@ -0,0 +1,258 @@
+// Type definitions for react-leaflet 1.0
+// Project: https://github.com/PaulLeCam/react-leaflet
+// Definitions by: Dave Leaver <https://github.com/danzel>
+// Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+// TypeScript Version: 2.1
+
+//tslint:disable:no-empty-interface
+
+/// <reference types="geojson" />
+
+import * as Leaflet from 'leaflet';
+import * as React from 'react';
+
+//All events need to be lowercase so they don't collide with React.DOMAttributes<T>
```

Comment:
If the event names aren't accurate to the names the library actually uses, there's no point...